src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -203,7 +203,7 @@ public static function singularify($plural)
         }
 
         // Convert teeth to tooth, feet to foot
-        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && 'feedback' !== $plural) {
+        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 && !preg_match('{feedback|Feedback|career|Career}', $plural)) {
```

Comment:
The regexp is not anchored, is it intended?
Using `in_array()` would be better don't you think?